timeseries/src/autogluon/timeseries/trainer/ensemble_composer.py

```diff
@@ -409,9 +409,19 @@ def _get_ensemble_model_name(self, name: str, layer_idx: int) -> str:
             name = f"{base_name}_{increment}" + layer_suffix
         return name
 
-    def _calculate_base_models_predict_time(self, model_names: list[str]) -> float:
+    def _calculate_predict_time(self, model: AbstractTimeSeriesEnsembleModel) -> float:
         """Calculate ensemble predict time as sum of base model predict times."""
-        return sum(self.model_graph.nodes[name]["predict_time"] for name in model_names)
+        assert model.predict_time_marginal is not None
+        predict_time = model.predict_time_marginal or 0
```

Comment:
After the assertion above, can we keep this as `predict_time = model.predict_time_marginal`?